django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -80,6 +80,8 @@ class SpatiaLiteOperations(BaseSpatialOperations, DatabaseOperations):
         'distance_gte': SpatialOperator(func='Distance', op='>='),
         'distance_lt': SpatialOperator(func='Distance', op='<'),
         'distance_lte': SpatialOperator(func='Distance', op='<='),
+
+        'isvalid': SpatialOperator(func='IsValid'),
```

Comment:
Is there a logical grouping with regards to the blank lines in this dict? If so, could you replace the blank lines with comments?